eslint.config.js

```diff
@@ -0,0 +1,53 @@
+import js from "@eslint/js";
+import prettier from "./node_modules/eslint-config-prettier/prettier.js";
```

Comment:
Nit: 
I know we import straight from node_modules for gts, we may be able to use the named import instead of the path import for prettier though.

Additionally, we should use eslintConfigPrettier

https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#installation

Reply:
OK, I think I followed the example from that link.